BSB_LAN/BSB_LAN.ino

```diff
@@ -888,7 +888,7 @@ void checkSockStatus()
     SPI.endTransaction();
 
     if ((s == 0x14) || (s == 0x1C)) {
-        if (thisTime - connectTime[i] > 30000UL) {
+        if (thisTime - connectTime[i] > 30000UL || connectTime[i] > thisTime) {
```

Comment:
@fredlcore Can you review fixes for timers when will you have free time?

Reply:
Sorry, I can't because I'm only using ESP32/Olimex here, and also the whole checkSockStatus never really worked for me, I think it was a suggestion by a user. Why are these changes necessary and why should there be any problems from it?

Reply:
When MCU will work more than 49 days millis() will overflow (because it return unsigned long) and starts from 0. And difference between thisTime and connectTime[i] will be always less 30000.
P.S. I will mark other lines with same problem.

Reply:
Yes, it is

> Used here is a unsigned long-variable which is a 4 byte variable.
Maximum value of uint32_t is (2^32)-1.
So when we take another of the same datatype, save 1 to it and deduct 4294967295 it first goes down to zero and then it "overflows" and roll over to its maximum value (((2^32)-1)) which it uses to go down from. So when it is finished deducting it ends up down at 2 :)

So here my mistake. I will revert this change. Thank you.

Reply:
Great, thanks, so this will apply to the other millis() sections below as well?

Reply:
Yep.